activerecord/lib/active_record/railties/databases.rake

```diff
@@ -501,6 +545,61 @@ db_namespace = namespace :db do
         db_namespace["test:load"].invoke
       end
     end
+
+    ActiveRecord::Tasks::DatabaseTasks.for_each(databases) do |spec_name|
+      # desc "Recreate the #{spec_name} test database"
```

Comment:
The descriptions on the tasks these are based on are _also_ commented out, so I kept that the same. Happy to change this as-needed.